vulture_os/toolkit/api_parser/harfanglab/harfanglab.py

```diff
@@ -127,8 +127,14 @@ def test(self):
     def get_logs(self, since=None, to=None, index=0):
         alert_url = f"{self.harfanglab_host}/{self.ALERTS}"
         # Remove miliseconds - format not supported
-        since_utc = since.isoformat().split('.')[0] + "+00:00"
-        to_utc = to.isoformat().split('.')[0] + "+00:00"
+        since_utc = since.isoformat().split('.')[0]
+        to_utc = to.isoformat().split('.')[0]
```

Comment:
potential alternative:
```suggestion
        since_utc = since.replace(microsecond=0).isoformat()
        to_utc = to.replace(microsecond=0).isoformat()
```